cps/main.c

```diff
@@ -575,6 +570,27 @@ static int
 assign_cps_queue_ids(struct cps_config *cps_conf)
 {
 	int ret;
+	/* Take the packets created for processing requests from mailbox. */
+	unsigned int total_pkt_burst = cps_conf->total_pkt_burst +
+		cps_conf->mailbox_burst_size;
+	unsigned int num_mbuf;
+
+	/* The front NIC doesn't have hardware support. */
+	if (!cps_conf->net->front.rss)
+		total_pkt_burst -= 2 * cps_conf->front_max_pkt_burst;
```

Comment:
Shouldn't we subtract just once since the other `cps_conf->front_max_pkt_burst` is still needed for the KNI?

If so, this comment also applies to the `if` statement below.